plugins/inputs/sqlserver/azuresqldbqueries.go

```diff
@@ -233,13 +233,27 @@ SELECT
 		WHEN slo.[edition] = 'Hyperscale' then NULL
 		ELSE CAST(DATABASEPROPERTYEX(DB_NAME(),'MaxSizeInBytes') as bigint)/(1024*1024)
 	END AS [total_storage_mb]
+	,CASE
+		WHEN slo.[edition] = 'Hyperscale' then NULL
+		ELSE (SELECT SUM(CAST(FILEPROPERTY(name, 'SpaceUsed') AS INT) / 128) FROM sys.database_files WHERE type_desc = 'ROWS')
+	 END AS used_space_data_mb
+	 ,CASE
+		WHEN slo.[edition] = 'Hyperscale' then NULL
+		ELSE (SELECT total_log_size_in_bytes / (1024 * 1024) FROM sys.dm_db_log_space_usage)
```

Comment:
Using `total_log_size_in_bytes` from `sys.dm_db_log_space_usage` may be misleading because it will report the current size of log file on disk, rather than its maximum size. Database files can grow on demand up to their defined maximum size, which is exposed in the `max_size` column of `sys.database_files`.

My suggestion would be to consider log maximum size as `total_log_mb`, as more relevant in most monitoring scenarios.